go-apps/meep-metrics-engine/api/v2/swagger.yaml

```diff
@@ -241,6 +241,49 @@ paths:
         500:
           description: "Internal server error"
 definitions:
+  NetworkQueryParams:
+    type: "object"
+    properties:
+      tags:
+        type: "array"
+        description: "Tag names to match in query. Supported values:<br> <li>src:\
+          \ Source network element name <li>dest: Destination network element name"
+        items:
+          $ref: "#/definitions/Tag"
+      fields:
+        type: "array"
+        description: "Field names to return in query response. Supported values:<br>\
+          \ <li>lat: Round-trip latency (ms)<br> <li>ul: UL throughput from src to\
+          \ dest (Mbps) <li>dl: DL throughput from dest to src (Mbps) <li>ulos: UL\
+          \ packet loss from src to dest (%) <li>dlos: DL packet loss from dest to\
+          \ src (%)"
+        items:
+          type: "string"
+          description: "Queried value"
+          enum:
+          - "lat"
+          - "ul"
+          - "dl"
+          - "ulos"
+          - "dlos"
```

Comment:
I had discussed this previously with Mike and we agreed that ul & dl implicitly mean througput. ulos and dlos were chosen as the shortest possible strings to represent ul & dl packet loss. This will be well documented and can be updated if it causes issues.